static/app/views/replays/detail/tagPanel/index.tsx

```diff
@@ -7,12 +9,26 @@ import Placeholder from 'sentry/components/placeholder';
 import {useReplayContext} from 'sentry/components/replays/replayContext';
 import ReplayTagsTableRow from 'sentry/components/replays/replayTagsTableRow';
 import {t} from 'sentry/locale';
+import useOrganization from 'sentry/utils/useOrganization';
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
 import FluidPanel from 'sentry/views/replays/detail/layout/fluidPanel';
 
 function TagPanel() {
+  const organization = useOrganization();
   const {replay} = useReplayContext();
   const replayRecord = replay?.getReplay();
 
+  const generateUrl = useCallback(
+    (name: string, value: string) =>
+      ({
+        pathname: normalizeUrl(`/organizations/${organization.slug}/replays/`),
+        query: {
+          query: `${name}:"${value}"`,
```

Comment:
What happens when value is an array-like value? Do we have a url per value, or is it a single url?

Will all "tags" that appear in this section have queries that work? I feel like I remember having to do some query trickery  with using `tag[]` to wrap a tag key.

With some tags (release specifically), we have some semi-standard behaviors. For release we generally show a release hovercard, will that be something we want to do in the future?